src/components/v5/common/ActionSidebar/partials/forms/PaymentBuilderForm/partials/PaymentBuilderRecipientsField/hooks.tsx

```diff
@@ -114,7 +115,13 @@ export const useRecipientsFieldTableColumns = (
                   {},
                 );
 
-                const payouts = Object.values(summedPayouts);
+                const payouts: ExpenditurePayout[] = Object.entries(
+                  summedPayouts,
+                ).map(([tokenAddress, { amount }]) => ({
+                  tokenAddress,
+                  amount,
+                  isClaimed: false,
+                }));
```

Comment:
You are not using the results of the `reduce` anywhere else, so you could modify the `reduce` to do all the data transformation in one go, without separate `Object.entries` and `map`. 

It is out of scope of this PR but it's worth noting these aren't payouts anymore, you've transformed it to be a map of token address and the total amount of payouts for that token address, and then made it into an array of objects representing token addresses and their amounts.

Since it's then passed somewhere else, it creates confusion as someone could reasonably expect the `payouts` passed to `PaymentBuilderPayoutsTotal` to be payouts, and not this special purpose array of objects.

I think we need to avoid this intermediary data manipulation as much as possible. If you passed "real" payouts to `PaymentBuilderPayoutsTotal` it could perform the necessary transformation there, and that would help us achieve a few important points:
- separation of concerns (this hook doesn't need to know what `PaymentBuilderPayoutsTotal` does)
- removal of the confusing data structures
- reducing back and forth data manipulation (currently we convert ETH to WEI in this hook, then convert it back to ETH in `paymentBuilderPayoutsTotal`)